java/client/src/main/java/glide/api/commands/GenericCommands.java

```diff
@@ -72,4 +74,23 @@ public interface GenericCommands {
      * }</pre>
      */
     CompletableFuture<Boolean> move(String key, long dbIndex);
+
+    CompletableFuture<String[]> sort(String key, SortStandaloneOptions sortStandaloneOptions);
+
+    CompletableFuture<String[]> sort(
+            String key, SortOptions sortOptions, SortStandaloneOptions sortStandaloneOptions);
```

Comment:
Please add to PR description what comes to `SortOptions` and what - to `SortStandaloneOptions`